packages/d3fc-webgl/src/shaders/fragmentShaderSnippets.js

```diff
@@ -128,6 +135,10 @@ export const bar = {
     body: `
         float canFill = 1.0;
         float canStroke = 0.0;
+
+        vec4 defaultFillColor = vec4(0.60, 0.60, 0.60, 1.0);
+        gl_FragColor = (canFill * defaultFillColor) + ((1.0 - canFill) * gl_FragColor);
```

Comment:
Can we just assign the default colour directly to `gl_FragColor` here, so `gl_FragColor = vec4(0.60, 0.60, 0.60, 1.0);`

At this point we know that `canFill = 1.0` and `canStroke = 0.0` so we know that this will always evaluate to using `defaultFillColor`